client/src/components/productDetails/eachComponent/AddToCart.jsx

```diff
@@ -134,6 +221,24 @@ class AddToCart extends React.Component {
             <button className='favoriteButton' type='button' onClick={this.favorite.bind(this)}>★</button>
           </div>
         </form>
+
+        <div id="myCartModal" class="cartModal">
+          {/* Modal content */}
+          <div className="cartModalContent">
+            <b><p>The following items have been added to your shopping cart!</p></b>
+            <span className="closeCartModal">[x]</span>
+            <p className='cartInnerText'>Some text in the Modal..</p>
+            <div className='modalButtonscheckout'>
+              <div>
+                <button onClick={this.closeModal.bind(this)} className='proceedToCheckout'>Proceed to checkout</button>
```

Comment:
declare closeModal bind once in constructor and you can just call this.closeModal everywhere else.